# BlockPose.msg

geometry_msgs/PoseStamped pose
string block_type
